src/containers/pages/InterventionPlan/UpdatePlan/index.tsx

```diff
@@ -138,10 +138,11 @@ const UpdatePlan = (props: RouteComponentProps<RouteParams> & UpdatePlanProps) =
     planStatus === PlanStatus.DRAFT &&
     isFIOrDynamicFI(initialValues.interventionType) &&
     initialValues.fiReason === CASE_TRIGGERED;
+
   const hiddenFields =
     HIDE_PLAN_FORM_FIELDS_ON_EDIT.length > 0 &&
     hideFieldsOnPlanStatuses.includes(initialValues.status) &&
-    !isCaseTriggeredAndDraft
+    isFIOrDynamicFI(initialValues.interventionType)
```

Comment:
Seems like this is the only change.  Could you describe how it solves the issue?

Reply:
@moshthepitt this ensures this fields are hidden for all `FI` and `Dynamic FI` either `Routine` or `Case Triggered` plans as opposed to earlier version that was hiding the fields for `Routine`  and not `case triggered` in draft mode.

Reply:
> this ensures this fields are hidden for all FI and Dynamic FI either Routine or Case Triggered plans as opposed to earlier version that was hiding the fields for Routine and not case triggered in draft mode.

Would have been nice to have this in a commit message :)